sql/2020/22_HTTP_2/number_of_h2_pushed_resources_and_avg_bytes_by_content_type.sql

```diff
@@ -0,0 +1,32 @@
+# standardSQL
+# Number of HTTP/2 Pushed Resources and Average Bytes by Content type
+SELECT 
+  client,
+  content_type,
+  COUNT(DISTINCT page) AS num_pages,
+  APPROX_QUANTILES(num_requests, 1000)[OFFSET(500)] AS median_pushed_requests,
```

Comment:
Use the `percentile` field. See `number_of_h2_pushed_resources_and_avg_bytes.sql` for example.

Reply:
Updated the query by adding percentile field. Used `number_of_h2_pushed_resources_and_avg_bytes.sql` as example. Checked the results against `sample_data.requests` table and looks correct.